# DESCRIPTION: A patch to make sure the program uses the correct python
#              implementation.
# AUTHOR: Karl Linden (lilrc) <lilrc@users.sourceforge.net>
# STATUS: Not sent upstream. (Oct 9 2013)

diff -ur playitslowly-1.4.0.orig/bin/playitslowly playitslowly-1.4.0/bin/playitslowly
--- playitslowly-1.4.0.orig/bin/playitslowly	2013-10-09 21:38:00.355813574 +0200
+++ playitslowly-1.4.0/bin/playitslowly	2013-10-09 21:38:28.122329816 +0200
@@ -1,2 +1,2 @@
 #!/bin/sh
-exec python -m playitslowly.app "$@"
+exec python2.7 -m playitslowly.app "$@"
